pkg/admissioncontroller/webhooks/auth/seed/handler.go

```diff
@@ -112,7 +114,7 @@ func (h *handler) Handle(w http.ResponseWriter, r *http.Request) {
 	h.logger.V(1).Info("received request", keysAndValues...)
 
 	// Consult authorizer for result and write the response
-	ctx, cancel := context.WithTimeout(context.Background(), DecisionTimeout)
+	ctx, cancel := context.WithTimeout(h.ctx, DecisionTimeout)
```

Comment:
Why is a timeout passed here? Our implementation is ignoring the context:
https://github.com/gardener/gardener/blob/a349f713c6a96b51beddd5c77f805ebfd58ad3d2/pkg/admissioncontroller/webhooks/auth/seed/authorizer.go#L95
Basically the DecisionTimeout is obsolete here, or am I overlooking something?

Reply:
You're right. It's somehow obsolete for this particular implementation. 
However, we reuse the interface from the API server library and there might be more implementations or changes to the existing one in the future, so it makes sense to fulfill the interface contract from the caller's side and pass a proper context.